src/test/app/Flow_test.cpp

```diff
@@ -1269,6 +1285,7 @@ struct Flow_test : public beast::unit_test::suite
         testWithFeats(featureFlow);
         testWithFeats(featureFlow, fix1373);
         testWithFeats(featureFlow, fix1373, featureFlowCross);
+        testEmptyStrand({featureFlow, fix1373, featureFlowCross});
```

Comment:
You could consider adding
```
testEmptyStrand({featureFlow, fix1373});
```
since I expect `fix1373` to be enabled on the network before `FlowCross`.  But it's not that important.  I tried the case on own machine and it passed.